tools/flang2/flang2exe/llassem.cpp

```diff
@@ -1164,8 +1164,13 @@ assemble_end(void)
       tdefsym = find_ag(gname);
       typed = AG_TYPENAME(tdefsym);
       fprintf(ASMFIL, "%%struct%s = type < { %s } > \n", name, typed);
+#if defined (_WIN64)
```

Comment:
Maybe this should use `target_triple` instead of checking for a macro on the host?